CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+
+## Contribution Guide :wrench:
+
+- Fork it
+- Create a seperate branch
+- Work on that, Work can be
+  - **New Features**
+  - **BugFix**
+  - **Better Docs**
+  - **Adding of Tests**
+  - **Code Cleaning + Refactors**
+  - **If any other kind of Contribution, Raise a issue for that and it can be discus there**
+- Raise a PR
+
+
+### Commit Guide
+We are using [commitlint](https://commitlint.js.org/#/) for our commit conventions
+
+Our commit conventions follow this pattern
+```
+type(scope?): subject
+body?
+footer?
+
+
+# type can be
+
+build
+ci
+chore
+docs
+feat
+fix
+perf
+refactor
+revert
+style
+test
+
+
+# example => feat(blog): add comment section
+
+
+```
+
+
+
+
+## How to Install and Use Locally
```

Comment:
I guess this is not needed. Cause it is already in the main readme.

Reply:
Sure